### PR TITLE
Chore: updated text mesh pro version to 3.2.0-pre9

### PR DESCRIPTION
**Pull request by davidejensen**

### CODE REVIEW

File: Explorer/Assets/AddressableAssetsData/AssetGroups/Localization-Locales.asset [hunk 1]
```diff
@@ -23,6 +23,18 @@ MonoBehaviour:
     m_SerializedLabels:
     - Locale
     FlaggedDuringContentUpdateRestriction: 0
+  - m_GUID: f81b53a5818cc4be9ab33b5208f77a18
```
Comment (aixaCode) [inline Explorer/Assets/AddressableAssetsData/AssetGroups/Localization-Locales.asset:26]: Are we again adding the Spanish and Italian? Or is that accidental?

**Review by aixaCode — APPROVED** [review]
LGTM. We probably don't need the HDRP TextMeshPro shaders, but anyway those will not be included in the build if nothing references them

File: Explorer/Assets/AddressableAssetsData/AssetGroups/Localization-Locales.asset (hunk 1 above)
Comment (davidejensen) [reply Explorer/Assets/AddressableAssetsData/AssetGroups/Localization-Locales.asset:26]: oh wow, totally missed this, I think it auto added them, let me remove

Comment (davidejensen) [reply Explorer/Assets/AddressableAssetsData/AssetGroups/Localization-Locales.asset]: Done

**Pull request merged**